library/bignum_mod.h

```diff
@@ -173,7 +173,51 @@ void mbedtls_mpi_mod_modulus_free( mbedtls_mpi_mod_modulus *m );
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+/** Read public representation data stored in a buffer into a residue structure.
+ *
+ * The `mbedtls_mpi_mod_residue` and `mbedtls_mpi_mod_modulus` structures must
+ * be compatible. The data will be automatically converted into the appropriate
+ * representation based on the value of `m->int_rep field`.
```

Comment:
So this function does a conversion, unlike `mbedtls_mpi_mod_residue_setup` which expects its input to be in the representation expected for `m`. Please document this for `mbedtls_mpi_mod_residue_setup`.